.vsts-ci.yml

```diff
@@ -81,7 +80,6 @@ jobs:
         PYTHON_VERSION: 3.7
       sdist:
         TASK: sdist
-        PYTHON_VERSION: 3.5
```

Comment:
so what version of Python will we get in these jobs now? Will it just float?

Reply:
We have "default" values in each CI config which are applied if this field is empty for particular job:
https://github.com/microsoft/LightGBM/blob/164818b029b02253796c9b2fe6391c572608f345/.vsts-ci.yml#L10-L11
https://github.com/microsoft/LightGBM/blob/164818b029b02253796c9b2fe6391c572608f345/.travis.yml#L15-L17
https://github.com/microsoft/LightGBM/blob/164818b029b02253796c9b2fe6391c572608f345/.appveyor.yml#L5-L6

Reply:
ooooo I missed that, ok